src/core/column-type/email.column.js

```diff
@@ -20,7 +20,7 @@ class EmailColumnModel extends DataColumnModel {
 	constructor() {
 		super('email');
 
-		this.canEdit = false;
+		this.editorOptions.editTrigger = 'button';
```

Comment:
i think 'custom' will be better and default name? Button knows about template too much..